wiki/Installing-Oppia-Android.md

```diff
@@ -63,85 +341,183 @@ Please follow these steps to set up Oppia Android on your local machine.
+
+6. A file similar to the one below will be generated:
+
+   <img width="760" alt="Image" src="https://github.com/user-attachments/assets/5a26210f-865d-4048-b5a5-73379984bfb5" />
+
+7. In the above file, modify it so that it looks like below(copy and paste):
+   
+   ```
+   directories:
+      # Add the directories you want added as source here
+      # By default, we've added your entire workspace ('.')
+      .
+      
+   # Automatically includes all relevant targets under the 'directories' above
+   derive_targets_from_directories: false
+   
+   targets:
+   # If source code isn't resolving, add additional targets that compile it here
+      //:oppia_dev_binary
+      //app
+      //app/src/main/...
+      //domain/...
+      //model/...
+      //testing/...
+      //utility/...
+      //data/...
+      //scripts/...
+      
+   additional_languages:
+      # Uncomment any additional languages you want supported
+      # c
+      # dart
+      # kotlin
+      java
+      kotlin
+
+   # Please uncomment an android-SDK platform. Available SDKs are:
+      # android_sdk_platform: android-28
+      android_sdk_platform: android-29
+      # android_sdk_platform: android-30
+      # android_sdk_platform: android-31
+      # android_sdk_platform: android-32
+      # android_sdk_platform: android-33
+      android_sdk_platform: android-34
+   
+   shard_sync: true
+   sync_flags:
+      --local_ram_resources=HOST_RAM*.5
+      --discard_analysis_cache
+   
+   java_language_level: 11
+   ```
+   **Note**: Only enable Android-SDK platforms that you have installed per the [Prepare developer environment](#prepare-developer-environment) section above. Android-SDK platforms are useful for creating emulators to test your app with.
+
+8. Click **Create**, and allow the project to synchronize.
+
+9. Once sync has finished, you can now build and install the app on either a virtual or physical device. Bazel supports deploying to only one device at a time, so you can connect one device, or launch one emulator at a time.
+
+   You can run the project by using the Bazel plugin to set up run configurations for the target that you wish to build. This performs the same action as the commands below, but using the GUI to run the app might be more intuitive for some developers.
+
+   ![Screenshot 2025-02-26 at 22 16 32](https://github.com/user-attachments/assets/f3be5288-dc96-4079-bcb2-b514de81f899)
+
+   Edit configurations allows us to specify the run command:
+
+   ![Image](https://github.com/user-attachments/assets/7270b6b3-11ac-4b49-a39a-734e0437bb9e)
+
+     - Target **Name**, is helpful for identifying the target from the list on the left.
+     - **Target expression** requires a build target such as `//:oppia_dev_binary`.
+     - **The Bazel command** is `mobile-install`
+     - **Bazel flags** are optional.
+     - Select **Apply** and then **Close** or **Ok**.
+
+## Run the app from the command line
```

Comment:
As mentioned in another comment, but perhaps this should go in the user guide and this updated to point to that?

In fact, I wonder if a lot of the "here's important Bazel stuff you need to do often or understand" should go in the user guide as a basis, and this page can be kept more for setting everything up.

Reply:
Moved to the user guide and included the link above.